<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-05DODOB" failures="1" tests="1" name="RunnerPackage.RunnerClass" time="12.393" errors="0" timestamp="2025-03-31T00:23:54 IST" skipped="0">
  <testcase classname="RunnerPackage.RunnerClass" name="runScenario" time="12.393">
    <failure message="Custom failure message expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Custom failure message expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at StepDefinition.BookLoginstepDefinition.login_link_should_appear(BookLoginstepDefinition.java:26)
at âœ½.Login Link should appear(file:///C:/Users/New/eclipse-workspace/Cucumber/src/test/java/FeaturePackage/Login.feature:24)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- RunnerPackage.RunnerClass -->
